src/Uno.UWP/UI/Input/GestureRecognizer.Manipulation.cs

```diff
@@ -0,0 +1,283 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using Windows.Devices.Input;
+using Windows.Foundation;
+using Uno.Extensions;
+
+namespace Windows.UI.Input
+{
+	public partial class GestureRecognizer
+	{
+		private class Manipulation
+		{
+			private enum ManipulationStates
+			{
+				Starting = 1,
+				Started = 2,
+				// Inertia = 3
```

Comment:
Why ?

Reply:
Because gesture recognize doesn't support Inertia yet (it's one of its responsibilities), but I wanted to reserved the value.

Reply:
Comment added :)